pkg/controller/node/nodecontroller.go

```diff
@@ -373,26 +381,29 @@ func (nc *NodeController) Run() {
 		defer nc.evictorLock.Unlock()
 		for k := range nc.zonePodEvictor {
 			nc.zonePodEvictor[k].Try(func(value TimedValue) (bool, time.Duration) {
-				obj, exists, err := nc.nodeStore.GetByKey(value.Value)
+				message, _ := value.UID.(evictionMessage)
```

Comment:
@gmarek I agree completely with your comment, but can you suggest how to fix it?